readme.md

```diff
@@ -221,3 +224,5 @@ abide by its terms.
 [xss]: https://en.wikipedia.org/wiki/Cross-site_scripting
 
 [sanitize]: https://github.com/rehypejs/rehype-sanitize
+
+[slug]: https://github.com/rehypejs/rehype-slug
```

Comment:
Seems there’s an enter missing at the end of this line!

Reply:
My bad! I was just editing the md and didn't install deps/run tests. Fixing...